core/datainput.gms

```diff
@@ -533,8 +533,12 @@ $ENDIF.tech_earlyreti
 *SB* Time-dependent early retirement rates in Baseline scenarios
 $ifthen.Base_Cprice %carbonprice% == "none"
 $ifthen.Base_techpol %techpol% == "none"
-*** Allow no early retirement in future periods under baseline for China; disallowing early retirement of coal all regions runs into EUR infesibility under H12
-pm_regiEarlyRetiRate(t,"CHA","pc")$(t.val ge 2020) = 0;
+*** CG: Allow no early retirement in future periods under baseline for developing countries; disallowing early retirement of coal all regions runs into EUR infesibility under H12
```

Comment:
thanks for this general formulation. I would delete the 2. sentence as this only hold now and might confuse in future